Add live wallpaper switch back option

12fab4 opened the pull request:
This just adds the option to revert the live wallpaper after a configurable timeout (when music stopped playing for eg. 30s).
This is my first android/kotlin code, so if there are any issues, please tell me.

Disclosure: i asked ChatGPT some questions (primarily how to revert the wallpaper). No Code from the repo was given to the chatbot.

12fab4 commented:
> you don't see my question? Let me copy it again.
> 
> > but then the next line gets executed too right? But it's disabled, so there should be an additional condition next to  `!isPlaying` or am I missing something here?

Yeah, my bad :(, i did not see this, fixed it

knighthat commented:
hey, this pr's missing so many things. Latest commit adds the requirement without importing it, and you're comparing MutableState with an enum value.

did you test whether the user-set wallpaper return to that 

the word "disabled" is hard-coded. To add a string entry, please use `string-modifier` module located in `scripts/strings-modifier/`

finally, the enum is created to store a single Long, why don't make it a Long to store? With Long, we can make a slider with `SettingComponent.Slider`

---

My advice, please fix all problems above and force push a new set of commits.

---

Sorry if this comes out harshly, but I'm in the process of moving some settings to kmp module so seeing these unnecessary use of enum is triggering me so much

12fab4 commented:
Sorry for waiting so long and no offense taken. I had some issues with android studio, but after updating to the latest version these seem to be fixed. I addressed all points you mentioned as good as i could and tested the app.
The last painpoint i see is that the wallpaper will be reset to the default manufacturer's one (which i think is an improvement over just leaving it as the last played song).
